tensorflow/tools/dockerfiles/spec.yml

```diff
@@ -161,6 +191,107 @@ slice_sets:
               - CHECKOUT_TF_SRC=1
               - CHECKOUT_HOROVOD_SRC=1
 
+    ubuntu-1604-onednn:
```

Comment:
It looks like these ubuntu-XXXX-onednn and ubuntu-XXXX-devel-onednn are used in the same places, right? These entries can be moved into "ubuntu-onednn" and "ubuntu-devel-onednn". Then the `dockerfiles` entries above would just be `{ubuntu-onednn}{onednn-jupyter}` and `{ubuntu-devel-onednn}{onednn-jupyter}`, since the sections get combined in all possible ways (that's why the others only have a `{jupyter}` section; one of jupyter's entries is blank).

Reply:
I wondered about that. It'll help with cleaning this up too.
Let me try.

Reply:
All `onednn` slice sets combined into one and of course `devel` and `non-devel` remain seperate.